src/dataEntryApp/components/DateFormElement.js

```diff
@@ -37,6 +37,11 @@ export const DateTimeFormElement = ({
   uuid
 }) => {
   const { t } = useTranslation();
+  const formtype = fe.formElementGroup.form.formType;
+  let fieldwidth = "30%";
+  if (formtype === "ProgramEnrolment") {
```

Comment:
We can't do conditional checks like these as there is nothing special about ProgramEnrolment form that requires fieldWidth to be different than other forms.